### DISCUSSION TITLE
Videos not working using poketube.fun

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7598] [General] [answered]
Hello!

I've been using qutebrowser for a span of 3 (4?) days and I am enjoying the experience so far!

But i've found an issue that has been bothering me. Every single video I open in https://poketube.fun doesn't play, and by looking at the console I haven't found anything that errors out. And I want to ask whether I should open an issue here, since videos in youtube work flawlessly and I am confused if that is an issue with QtWebEngine or an issue with poketube itself.

Best regards,
StationaryStation :D

**Comment by toofar** [comment]
It doesn't even get as far as individual videos for me. It just redirects to https://poketube.fun/?fromerror=41 when I click on one, same for chrome...

Does it work on chrome or firefox for you? How about KDE falkon (another webengine based browser?).

**Reply by ghost** [reply]
It seems to not work in falkon as well...

**Comment by ghost** [comment]
I am going to try later, as I am about to go and take a bath. I did try on Vieb, another qute like browser based on electron, and it seemed to work fine, it was also working on Firefox and Chrome(ium)On 3 Mar 2023 19:40, toofar ***@***.***> wrote:
It doesn't even get as far as individual videos for me. It just redirects to https://poketube.fun/?fromerror=41 when I click on one, same for chrome...
Does it work on chrome or firefox for you? How about KDE falkon (another webengine based browser?).

**Comment by ghost** [comment, marked as answer]
Good News!

Apparently, installing `qt5-webengine-freeworld` fixes the issue and playback works as expected!

I've forgotten that fedora doesn't have some codecs in the `qt5-webengine` package and it seems that it was the root cause of the issue.
